### PR TITLE
add option to lib/Stats.js to disable stack trace on errors and warnings

### PR DESCRIPTION
**Pull request by deificx**
**What kind of change does this PR introduce?**

Adds an extra option to the Stats library to disable the "stack trace" produced in formatError

**Did you add tests for your changes?**

No

**If relevant, link to documentation update:**

Unsure where and how to update this, please point me in the right direction, thank you :-) 

**Summary**

After upgrading to webpack 2 I found the output to be a bit more verbose, as errors/warnings already can and often shows filename I don't find the stack trace to add anything other than clutter to the output. I searched around briefly for a solution to disable this but only found an unanswered stack overflow question (http://stackoverflow.com/questions/43403615/suppress-webpack-2-eslint-extra-output-stack-trace), so at least I'm not alone.

**Does this PR introduce a breaking change?**

No

### CODE REVIEW

File: lib/Stats.js [hunk 1]
```diff
@@ -95,6 +95,7 @@ class Stats {
 		const showProvidedExports = optionOrFallback(options.providedExports, !forToString);
 		const showChildren = optionOrFallback(options.children, true);
 		const showSource = optionOrFallback(options.source, !forToString);
+		const showStackTrace = optionOrFallback(options.stackTrace, true);
```
Comment (sokra) [inline lib/Stats.js]: Maybe we call it `moduleTrace` instead.

**Comment by sokra** [comment]
Thanks

**Comment by talon** [comment]
I LOVE YOU

**Comment by gaearon** [comment]
If anyone's interested, here's what we do in CRA to clean up Webpack output:

https://github.com/facebookincubator/create-react-app/blob/master/packages/react-dev-utils/formatWebpackMessages.js

Would love to have access to more structured errors so we can remove these hacks.

**Pull request merged**